<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Foodventory</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>0</y>
      <width>732</width>
      <height>480</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>381</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="font">
          <font>
           <pointsize>50</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TIME</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dateLabel">
         <property name="font">
          <font>
           <pointsize>35</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DATE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dayLabel">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DAY</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="scanButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>271</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SCAN</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>320</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="cityLabel">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="text">
          <string>City</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tempLabel">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Temp</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="weatherPhraseLabel">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Phrase</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="updateWeatherButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>20</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="inventory">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>667</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="invAddButton">
      <property name="geometry">
       <rect>
        <x>664</x>
        <y>-1</y>
        <width>71</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="invDeleteButton">
      <property name="geometry">
       <rect>
        <x>664</x>
        <y>97</y>
        <width>71</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="invSearchLineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>440</y>
        <width>272</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Search...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="invFindButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>439</y>
        <width>250</width>
        <height>43</height>
       </rect>
      </property>
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
     <widget class="QPushButton" name="invInstructionsButton">
      <property name="geometry">
       <rect>
        <x>664</x>
        <y>270</y>
        <width>71</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Instruct</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="list">
     <widget class="QLineEdit" name="listEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>New item...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>121</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>160</y>
        <width>121</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>290</y>
        <width>121</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QTableView" name="listTable">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>421</width>
        <height>390</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>370</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="zipEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>ZIP Code</string>
      </property>
     </widget>
     <widget class="QLabel" name="settingsLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="weatherLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Weather</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="zipLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>128</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ZIP Code:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fullscreenOnButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ON</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">fullscreenButtonGroup</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="fullscreenLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>199</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Fullscreen:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fullscreenOffButton">
      <property name="geometry">
       <rect>
        <x>228</x>
        <y>190</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">fullscreenButtonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="saveZipButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rotateCameraOnButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ON</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">rotateCameraButtonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="rotateCameraOffButton">
      <property name="geometry">
       <rect>
        <x>228</x>
        <y>250</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">rotateCameraButtonGroup</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="rotateCameraLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotate Camera:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="scanner">
     <widget class="QPushButton" name="manualButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>390</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Manually</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>640</width>
        <height>355</height>
       </rect>
      </property>
      <property name="text">
       <string>Loading Camera...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="manualEnter">
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="brandLineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Brand</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="upcLineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>UPC</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="manualAddButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>130</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manualCancelButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>250</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quantity</string>
      </property>
     </widget>
     <widget class="QPushButton" name="barcodeScanButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Barcode Scan</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="quantitySpinBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>110</y>
        <width>181</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QTextEdit" name="instructionsEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>391</width>
        <height>151</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Instructions...</string>
      </property>
     </widget>
     <widget class="QComboBox" name="categoryComboBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="locationComboBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>70</width>
      <height>123</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="invButton">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>119</y>
      <width>70</width>
      <height>122</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="listButton">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>238</y>
      <width>70</width>
      <height>123</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LIST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>358</y>
      <width>70</width>
      <height>124</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SETTINGS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="fullscreenButtonGroup"/>
  <buttongroup name="rotateCameraButtonGroup"/>
 </buttongroups>
</ui>
